Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py

```diff
@@ -354,16 +357,47 @@ def create_pull_request(cls, repository, args, branch_point, callback=None, unbl
 
         existing_pr = None
         if remote_repo.pull_requests:
+            user, _ = remote_repo.credentials(required=False)
+
             log.info("Checking if PR already exists...")
             existing_pr = cls.find_existing_pull_request(repository, remote_repo)
```

Comment:
If I take over a PR (by pushing to the author's branch), will this call notice my coauthorship, or will I have to go through the "Would you like to take over this PR?" prompt again?

Reply:
It should assign the PR to you, so you won't get the prompt.

When working on this PR, I discovered that most contributors probably haven't given others write access to their branches, so if you fail to push, we won't update the PR and then you'll get the prompt again, but that seems appropriate.